src/compute-plots.jl

```diff
@@ -389,18 +425,41 @@ function add_theme!(plot::T, scene::Scene) where {T}
     return
 end
 
-register_camera!(scene::Scene, plot::Plot) = register_camera!(plot.args[1], scene.compute)
+register_camera!(scene::Scene, plot::Plot) = register_camera!(plot.attributes, scene.compute)
 
-function connect_plot!(parent::SceneLike, plot::ComputePlots)
-    computed_plot!(parent, plot)
+function Plot{Func}(user_args::Tuple, user_attributes::Dict) where {Func}
+    # Handle plot!(plot, attributes::Attributes, args...) here
+    if !isempty(user_args) && first(user_args) isa Attributes
+        attr = attributes(first(user_args))
+        merge!(user_attributes, attr)
+        return Plot{Func}(Base.tail(user_args), user_attributes)
+    end
+
+    P = Plot{Func}
+    attr = ComputeGraph()
+    add_attributes!(P, attr, user_attributes)
+    register_arguments!(P, attr, user_attributes, user_args)
+    converted = attr[:converted][]
+    ArgTyp = MakieCore.argtypes(converted)
+    FinalPlotFunc = plotfunc(plottype(P, converted...))
+    p = Plot{FinalPlotFunc,ArgTyp}(user_attributes, attr)
+    p.transformation = Transformation()
```

Comment:
`handle_transformation!()` will replace `p.transformation` so maybe it's better to leave it undefined here? That way trying to do anything will fail instead of seemingly working